### ISSUE TITLE
Users unable to login to Premium search in Private windows

### ISSUE THREAD

**Issue by Brave-Matt** [brave/brave-browser#56790]
### Description

On desktop and Android builds, logging into account.brave.com and selecting `Open` for Premium search will launch a new search tab, logged in as a Premium user. This however does not work on iOS. While you can log into account.brave.com, clicking `Open` does not launch the site and detect that it's a Premium subscription. 

### Steps to reproduce

1. Open Private tab on iOS
2. Login to account.brave.com
3. Attempt to launch Premium search session 

### Actual result

Premium search session does not start

### Expected result

User's should be able to use Premium search subscription in Private browsing mode

### Reproduces how often

Easily reproduced

### Brave version

1.90.2

### Device/iOS version

All

### Affected browser versions

- [x] latest AppStore
- [ ] latest TestFlight
- [ ] previous TestFlight

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] in the latest version of mobile Safari

**Comment by kjozwiak** [comment]
The above requires `1.93.121` or higher for `1.93.x` verification(s) 👍

**Comment by Uni-verse** [comment]
Verified on `iPhone 15` running `iOS 26.5` using version `1.93 (131)`

- Ensured that premium search session is started when launching from `account.brave.com` as outlined in the STR in https://github.com/brave/brave-browser/issues/56790#issue-4780355002
- Ensured that starting premium search works in private and normal tabs. 

**`Normal Tab`** | **`Private Tab`**
--|--
<img width="590" height="1278" alt="Image" src="https://github.com/user-attachments/assets/76385624-6f21-4469-b3ab-cd257a34cd6d" /> | <img width="590" height="1278" alt="Image" src="https://github.com/user-attachments/assets/21d99363-285d-4c08-86c2-af1e2bcb3d4c" />
<img width="590" height="1278" alt="Image" src="https://github.com/user-attachments/assets/a0d87315-4bb5-4520-9a47-906f6d6c294c" /> | <img width="590" height="1278" alt="Image" src="https://github.com/user-attachments/assets/2aa0d40f-36f5-49c4-8523-a9db0e4f35fd" />
